### PR TITLE
Update FDC link for MVP and v2

### PR DESCRIPTION
**Pull request by markgreenburg**
## Description
Updates FDC link for v2 and mvp of 526 form.

## Testing done
- Tested the links locally
- Unit tests

## Screenshots
This link:
screen shot 2019-01-28 at 11 49 12 am

Goes to:
screen shot 2019-01-28 at 11 49 17 am

## Acceptance criteria
- [x] Update FDC links on mvp and v2

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/disability-benefits/all-claims/content/fullyDevelopedClaim.jsx [hunk 1]
```diff
@@ -8,7 +8,10 @@ export const FDCDescription = (
       uploaded all the supporting documents or supplemental forms needed to
       support your claim.
     </p>
-    <a href="/pension/how-to-apply/fully-developed-claim/" target="_blank">
+    <a
+      href="/disability/how-to-file-claim/evidence-needed/fully-developed-claims/"
+      target="_blank"
```
Comment (cvalarida) [inline src/applications/disability-benefits/all-claims/content/fullyDevelopedClaim.jsx]: Should this also have the `rel` per your other PR?

**Review by cvalarida — APPROVED** [review]

**Review by peggygannon — APPROVED** [review]
@markgreenburg Looks good. thanks for updating so quickly!

File: src/applications/disability-benefits/all-claims/content/fullyDevelopedClaim.jsx (hunk 1 above)
Comment (markgreenburg) [reply src/applications/disability-benefits/all-claims/content/fullyDevelopedClaim.jsx]: 🤔 I don't think so, this seems to not be a teamsite link

**Review by bshyong — APPROVED** [review]
LGTM

**Pull request merged**